src/test/regress/expected/multi_mx_modifications.out

```diff
@@ -426,28 +426,29 @@ SELECT minimum_value::bigint AS min_value,
   FROM information_schema.sequences
   WHERE sequence_name = 'app_analytics_events_mx_id_seq' \gset
 SELECT last_value FROM app_analytics_events_mx_id_seq \gset
+SET citus.enable_ddl_propagation TO OFF;
```

Comment:
I'd rather `RESET` back right after `ALTER SEQUENCE`. Is there a specific reason for not doing that?